src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/ITagHelper.cs

```diff
@@ -25,7 +25,7 @@ public interface ITagHelper
         /// </summary>
         /// <param name="context">Contains information associated with the current HTML tag.</param>
         /// <remarks>When more than one <see cref="ITagHelper"/> runs on the same element,
-        /// <see cref="TagHelperOutput.GetChildContentAsync"/> may be invoked prior to <see cref="ProcessAsync"/>.
+        /// <see cref="TagHelperOutput.GetChildContentAsync()"/> may be invoked prior to <see cref="ProcessAsync"/>.
```

Comment:
Suggest filing a bug (w/ a minimal repro) on Roslyn, an External bug to track it, and a Razor bug to fix this comment eventually.